chain/consensus/filcns/upgrades.go

```diff
@@ -1506,12 +1503,8 @@ func LiteMigration(ctx context.Context, bstore blockstore.Blockstore, newActorsM
 	// Maps prior version code CIDs to migration functions.
 	migrations := make(map[cid.Cid]cid.Cid)
 
-	for _, entry := range newManifestData.Entries {
-		oldCodeCid, ok := oldManifest.Get(entry.Name)
-		if !ok {
-			return cid.Undef, xerrors.Errorf("code cid for %s actor not found in old manifest", entry.Name)
-		}
-		migrations[oldCodeCid] = entry.Code
+	for i, entry := range newManifestData.Entries {
+		migrations[oldManifestData.Entries[i].Code] = entry.Code
```

Comment:
This assumes that we aren't adding/removing any actors. We need to lookup actors by name.

Reply:
Yeah, you're right, i got carried away with my desperation to simplify things.